assume/strategies/naive_strategies.py

```diff
@@ -158,9 +176,9 @@ def calculate_bids(
             and the volume of the product. Dispatch the order to the market.
             """
             start = product[0]
-
             volume = unit.opt_power_requirement.loc[start]
             marginal_price = unit.calculate_marginal_cost(start, volume)
+
             bids.append(
                 {
```

Comment:
Is a unit`s node relevant within the bidding workflow? If yes maybe it needs to be added here.

Reply:
for the naive strategy on DA it is not relevant